cmd/xyz2svg/cmd/draw.go

```diff
@@ -175,7 +175,7 @@ func drawFeatures(pname string, tiler provider.Tiler, layers []string, gid int,
 
 			// Clip and validate
 			ext := geom.NewExtent([2]float64{pbb[0], pbb[1]}, [2]float64{pbb[2], pbb[3]})
-			vg, err := validate.CleanGeometry(ctx, sg, ext)
+			vg, err := validate.CleanGeometry(ctx, sg, ext, true)
```

Comment:
How about instead of adding in a new parameter we just pass in a nil extent? The Extent methods are designed to check for nil. If we do this, I think we can clean up a bunch of the additions in this PR.